sessions/recap-project-2/example-implementation/profile.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <script src="profile.js" defer></script>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__title">Profile</h1>
+    </header>
+    <main class="profile">
+      <div class="profile__card">
+        <img
+          class="profile__card-img"
+          src="https://source.unsplash.com/random/300x300?portrait"
+          alt=""
+          height="200"
+          width="200"
+        />
+        <h2 class="profile-card__name">Max Doe</h2>
+      </div>
+      <h3>About me</h3>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatum
+        dignissimos, reprehenderit quam facilis animi in distinctio nemo est
+        culpa velit non unde expedita porro. Expedita animi quo voluptates
+        blanditiis libero?
+      </p>
+      <ul class="counter-list">
+        <li class="counter-list__item">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            height="24"
+            width="24"
+            viewbox="0 0 48 48"
+            fill="currentColor"
+          >
+            <path
+              d="M21.55 31.5q.05-3.6.825-5.25.775-1.65 2.925-3.6 2.1-1.9 3.225-3.525t1.125-3.475q0-2.25-1.5-3.75t-4.2-1.5q-2.6 0-4 1.475T17.9 14.95l-4.2-1.85q1.1-2.95 3.725-5.025T23.95 6q5 0 7.7 2.775t2.7 6.675q0 2.4-1.025 4.35-1.025 1.95-3.275 4.1-2.45 2.35-2.95 3.6t-.55 4Zm2.4 12.5q-1.45 0-2.475-1.025Q20.45 41.95 20.45 40.5q0-1.45 1.025-2.475Q22.5 37 23.95 37q1.45 0 2.475 1.025Q27.45 39.05 27.45 40.5q0 1.45-1.025 2.475Q25.4 44 23.95 44Z"
+            />
+          </svg>
+          <span>12</span>
+        </li>
+        <li class="counter-list__item">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewbox="0 0 24 24"
+            width="24px"
+            fill="currentColor"
+          >
+            <path d="M0 0h24v24H0V0z" fill="none" />
+            <path d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2z" />
+          </svg>
+          <span>4</span>
```

Comment:
Do we need this `span`? 🤔 

```suggestion
          4
```